api/src/test/java/org/openmrs/module/pihcore/status/StatusDataFunctionsTest.java

```diff
@@ -19,20 +19,30 @@ public void testVelocityExpressions() {
         Calendar cal = Calendar.getInstance();
         DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
         String today = df.format(cal.getTime());
+
+        cal = Calendar.getInstance();
         cal.add(Calendar.YEAR, 3);
         dataValues.put("in3Years", cal.getTime());
-        cal.add(Calendar.YEAR, -3);
+
+        cal = Calendar.getInstance();
         cal.add(Calendar.DATE, 52);
         dataValues.put("in52Days", cal.getTime());
-        cal.add(Calendar.DATE, -52);
+
+        cal = Calendar.getInstance();
         cal.add(Calendar.DATE, -1);
         dataValues.put("yesterday", cal.getTime());
         String yesterday = df.format(cal.getTime());
-        cal.add(Calendar.DATE, -7);
```

Comment:
Note that this was subtracting 7 days from yesterday, but I changed it to subtract a week from today.